spec/requests/portfolios_spec.rb

```diff
@@ -0,0 +1,135 @@
+describe 'Portfolios API' do
+  include RequestSpecHelper
+
+  let!(:portfolio)            { create(:portfolio) }
+  let!(:portfolio_item)       { create(:portfolio_item) }
+  let!(:portfolio_items)      { create_list(:portfolio_item, 3, :portfolios => [portfolio]) }
+  let(:portfolio_id)          { portfolio.id }
+  let(:portfolio_item_id)     { portfolio_items.first.id }
+  let(:new_portfolio_item_id) { portfolio_item.id }
+  let(:tenant)                { create(:tenant, :with_external_tenant) }
+
+  # Encoded Header: { 'identity' => { 'is_org_admin' => false, 'org_id' => 111 } }
```

Comment:
@syncrou These could go in here
https://github.com/ManageIQ/service_portal-api/blob/ccca63a2289b43874e280f6c95ff9c25e98d8281/spec/support/service_spec_helper.rb#L6